### ISSUE TITLE
load checkpoint from local path: checkpoints/RealBasicVSR_x4.pth Killed

### ISSUE THREAD

**Issue by GusRoth** [ckkelvinchan/RealBasicVSR#52]
python inference_realbasicvsr.py configs/realbasicvsr_x4.py checkpoints/RealBasicVSR_x4.pth data/demp2.mp4 results/demo_001.mp4 --fps=59
2022-06-02 13:27:00,288 - mmedit - INFO - load checkpoint from torchvision path: torchvision://vgg19
load checkpoint from local path: checkpoints/RealBasicVSR_x4.pth
Killed

why?

it even not use my memory and cpu ,gpu

I run it in wsl2 which support cuda

**Comment by ckkelvinchan** [comment]
I did not encounter this error before, I think this is related to MMEditing. Maybe you can raise an issue [there](https://github.com/open-mmlab/mmediting)

**Comment by LiBai3121** [comment]
I have encounter the same error. Have you fixed it yet? Thanks

**Comment by IceWind233** [comment]
Obviously, i encounted same problem and i raise [issue](https://github.com/open-mmlab/mmagic/issues/2107)

**Comment by ZhangFelton** [comment]
same issue! Any solutions?
